napari/_qt/_qapp_model/_tests/test_qdispatcher.py

```diff
@@ -0,0 +1,140 @@
+import time
+from unittest.mock import MagicMock, patch
+
+from app_model.registries import CommandsRegistry
+
+from napari._qt._qapp_model._qdispatcher import (
+    PRESS_HOLD_DELAY_MS,
+    QKeyBindingDispatcher,
+)
+from napari.utils.key_bindings import DispatchFlags
+
+
+def test_q_key_binding_dispatcher(qtbot):
+    with patch.object(QKeyBindingDispatcher, 'executeCommand'):
```

Comment:
As patch is used for the whole test it should be done using either decorator:

```python
@patch.object(QKeyBindingDispatcher, 'executeCommand')
def test_q_key_binding_dispatcher(_patch, qtbot):
    ...
```

or using monkeypatch

```python
def test_q_key_binding_dispatcher(monkeypatch, qtbot):
    monkeypatch.setattr(QKeyBindingDispatcher, 'executeCommand' , MagicMock())
    ...
```

Both remove obsolete indent that reduces readability. 

If the split function on multiple, then custom fixture could be used:


```python 
@pytest.fixture
def patch_exc_com(monkeypatch):
    monkeypatch.setattr(QKeyBindingDispatcher, 'executeCommand' , MagicMock())

@pytest.mark.usefixture("patch_exc_com")
def test_q_key_binding_dispatcher(qtbot):
```

or 

```python 
@pytest.fixture
def patch_exc_com():
    with patch.object(QKeyBindingDispatcher, 'executeCommand'):
        yield

@pytest.mark.usefixture("patch_exc_com")
def test_q_key_binding_dispatcher(qtbot):
```